crates/rpc/rpc/src/eth/signer.rs

```diff
@@ -34,6 +41,24 @@ pub(crate) struct DevSigner {
     accounts: HashMap<Address, SecretKey>,
 }
 
+impl DevSigner {
+    fn sign_hash(&self, hash: H256, account: Address) -> Result<Signature> {
+        let secp = Secp256k1::new();
```

Comment:
use global

Reply:
Not sure if this is what you had in mind, but I've added a field to DevSigner that contains a `Secp256k1` :slightly_smiling_face: